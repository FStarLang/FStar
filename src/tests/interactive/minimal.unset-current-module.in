{"query-id":"1","query":"push","args":{"kind":"full","code":"module Minimal\n","line":1,"column":0}}
{"query-id":"2","query":"pop","args":{"dummy":null}}
{"query-id":"3","query":"lookup","args":{"symbol":"hello","requested-info":["type","documentation"],"location":{"filename":"<input>","line":2,"column":4}}}
