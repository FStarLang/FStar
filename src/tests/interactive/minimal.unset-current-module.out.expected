{"features": ["autocomplete", "compute", "describe-protocol", "describe-repl", "exit", "lookup", "lookup/documentation", "lookup/definition", "pop", "peek", "push", "search"], "kind": "protocol-info", "version": 2}
{"kind": "response", "query-id": "1", "response": [], "status": "success"}
{"kind": "response", "query-id": "2", "response": null, "status": "success"}
{"kind": "response", "query-id": "3", "response": "Current module unset", "status": "failure"}
